Documentation/commands.md

```diff
@@ -110,7 +110,7 @@ sha512-fa1cb92dc276b0f9bedf87981e61ecde
 If you haven't trusted the creator, it will be downloaded but not verified:
 
 ```
-$ sudo ./rkt fetch coreos.com/etcd:v2.0.0
+$ rkt fetch coreos.com/etcd:v2.0.0
```

Comment:
Yeah, we should be able to fetch as a normal user so that's why I didn't change it but I guess it's better to use the root shell here to not complicate things.